tools/clang/test/CodeGenSPIRV/method.append-structured-buffer.get-dimensions.hlsl

```diff
@@ -0,0 +1,13 @@
+// Run: %dxc -T vs_6_0 -E main
+
+AppendStructuredBuffer<float4> buffer;
+
+void main() {
+  uint numStructs, stride;
+  
+// CHECK:      [[buf:%\d+]] = OpLoad %type_AppendStructuredBuffer_v4float %buffer
+// CHECK-NEXT: [[len:%\d+]] = OpArrayLength %uint [[buf]] 0
+// CHECK-NEXT: OpStore %numStructs [[len]]
+// CHECK-NEXT: OpStore %stride %uint_16
```

Comment:
I'd suggest to parameterize `AppendStructuredBuffer` with some struct containing a few fields so that we can indeed check the `stride` is good. `float4` is too simple IMHO.